PeerioTechnologies/peerio-mobile
# feat: retry login 3 times before failing [ch18095]

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/18095/mobile-automatic-retry-on-failed-login-when-logging-through-loading-screen

#### Testing instructions  
One comparatively easy way to test this is:
- creating a user on server 1
- disabling the ability of user to login
- starting the app again

The logs should contain the following entries:
`retry login: failed ${i} time out of ${count}`;
`failing automatic login after ${i} tries`;

indicating that there was a retry.

#### Possible regressions:
Login on return (with previously authenticated users)


----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
